eventmesh-storage-plugin/eventmesh-storage-kafka/src/main/java/org/apache/eventmesh/storage/kafka/producer/ProducerImpl.java

```diff
@@ -107,7 +107,7 @@ public void sendOneway(CloudEvent message) {
 
     public void sendAsync(CloudEvent cloudEvent, SendCallback sendCallback) {
         try {
-            this.producer.send(new ProducerRecord<>(cloudEvent.getSubject(), cloudEvent), (metadata, exception) -> {
+            this.producer.send(new ProducerRecord<>(Objects.requireNonNull(cloudEvent.getSubject(), cloudEvent), (metadata, exception) -> {
```

Comment:
![image](https://user-images.githubusercontent.com/91315508/236098658-b2212e3f-f715-4e54-8b53-431090f1cf33.png)
Need to add a `)`.